tutorials/migration_hydrodynamics_addedmass.md

```diff
@@ -0,0 +1,85 @@
+# Migrating Added Mass from the Hydrodynamics to the Fluid Added Mass tag
+
+Historically, we used the specify added mass in the parameters of the hydrodynamics plugin.
+Starting with GazeboIonic, we have deprecated that in favour of using the SDF `<fluid_added_mass>` tag directly. This guarantees better numerical stability. That being said, in order to actually simulate added mass your underlying physics engine must be able to support the added mass tag. So far that only includes the DART physics engine.
```

Comment:
```suggestion
Starting with Gazebo Ionic, we have deprecated that in favour of using the SDF `<fluid_added_mass>` tag directly. This guarantees better numerical stability. That being said, in order to actually simulate added mass your underlying physics engine must be able to support the added mass tag. So far that only includes the DART physics engine.
```

Reply:
Thanks for spotting this. Resolved in 6a235ee810b5e6f857309caf8a0e7af2f3ae23dc